src/components/features/widgets/map/MapWithHeaderSection/index.tsx

```
@@ -1,29 +1,20 @@
 "use client";
 
-import Spinner from "@/components/common/shared/ui/Spinner";
+import Map from "@/components/common/entities/Map";
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)